### DISCUSSION TITLE
I do not see credentials on any MDF's. How do I login to development instance?

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#3487] [Q&A] [answered]
Hi Team,
I was going through the HACKING.MD and BUILDING.MD file to setup my dev env. Although I am done with what's mentioned, I am stuck on what credentials (email and password) to use to access developer instance / sandbox.

Please don't tell me we need to sign up and use our personal email address 🤞🏼😅

**Comment by charlag** [comment, marked as answer]
Hi
Sorry but we don't have a public sandboxed environment right now

**Reply by ghost** [reply]
@charlag ... So, if we are entering our personal email address for this, are we testing application directly on production server?

Also, What is the workflow of `npm test` like?
	When i issue the command its stuck ..

	Connected to the build server
	Server: New build request with parameters: {}
	Server: Config has changed, rebuilding old: undefined, new: {}
	Server: Builder: Building tests

**Reply by charlag** [reply]
@prnam if you build for `prod` then yes it's your email.

Is it really stuck or does it just build it?

**Reply by ghost** [reply]
Nothing happens after that line, @charlag. 

In my opinion, It would be of great help if there was GIF / screenshot as well on MDF's.
